packages/language-support/src/tests/benchmarks/benchmark.ts

```diff
@@ -18,21 +16,6 @@ Benchmark.options.minSamples = 20;
 const suite = new Benchmark.Suite();
 
 suite
-  .add('simple - parse', function () {
-    parse(simpleQuery);
-  })
-  .add('simple - highlight', function () {
-    parserWrapper.clearCache();
-    applySyntaxColouring(simpleQuery);
-  })
-  .add('simple - validate syntax', function () {
-    parserWrapper.clearCache();
-    validateSyntax(simpleQuery, benchmarkingMediumSizeSchema);
-  })
-  .add('simple - autocomplete next statement', function () {
-    parserWrapper.clearCache();
-    autocomplete(simpleQuery, benchmarkingMediumSizeSchema);
-  })
```

Comment:
Why are we removing benchmarks? Wouldn't it be better to leave them but just run them on merge to `main`?

Reply:
I was in the "reduce gh minute" mindset, so I removed some that didn't feel like they were super impactful, to see if the build got noticeably faster. But you're right, I think we're better off keeping them - i added them back now